bin/compile

```diff
@@ -70,10 +70,10 @@ if [ -f "$RUNTIME_FILE" ] ; then
     log "$RUNTIME_FILE found, skipping $RUNTIME_FILE generation" >&2
     exit 0
 else
-    log "Generating $RUNTIME_FILE from uv Python version"
+    log "Generating .python_version from uv Python version"
     PYTHON_VERSION_OUTPUT=$(uv run python --version)
-    echo "${PYTHON_VERSION_OUTPUT//Python /python-}" > "$RUNTIME_FILE"
```

Comment:
The `/python-` here was important to strip it down to just the version number, wasn't it? What does your cat show for the final file contents? Thanks!

Reply:
it replaces `'Python '` with `''`. i.e. `Python 3.14.15` → `3.14.15`

Reply:
Whoops! Sorry, looks good to me!